sphinx/source/docs/user_guide/interaction/js_callbacks.rst

```diff
@@ -72,33 +121,35 @@ any Bokeh model, using the ``js_on_change`` method of Bokeh models:
     # execute a callback whenever p.x_range.start changes
     p.x_range.js_on_change('start', callback)
 
-It should be mentioned that the first parameter to ``js_on_change`` is
-actually the name of a BokehJS event. The full format for a property
-change event is, for example, ``"change:start"``, but Bokeh will automatically
-convert any property name into one of these BokehJS change events for you.
-Additionally, some Bokeh models have additional specialized events. For
-example, the ``ColumnDataSource`` also supports ``"patch"`` and ``"stream"``
-events, for executing ``CustomJS`` callbacks whenever the data source is
-patched or streamed to.
+The first parameter to ``js_on_change`` is actually the name of a BokehJS event.
+The full format for a property change event is, for example, ``"change:start"``.
+However, Bokeh will automatically convert any property name into one of these
+BokehJS change events for you.
+
+Some Bokeh models have additional, specialized events. For example, the
+:class:`~bokeh.models.ColumnDataSource` model also supports ``"patch"`` and
+``"stream"`` events. You can use these events to trigger ``CustomJS`` callbacks
+whenever the data source is patched or streamed to.
```

Comment:
I wonder if this is too much information, we don't demonstrate the `"change:foo"` usage anywhere really, and I am not really sure we should advertise it.